schema/stateTransition/base.json

```diff
@@ -8,6 +8,30 @@
     "type": {
       "type": "number",
       "enum": [1, 2]
+    },
+    "publicKeyId": {
```

Comment:
Should it be valid without signature

Reply:
Remove signaturePublicKey id from the required fields